app/src/main/java/org/literacyapp/authentication/helper/AuthenticationInstructionHelper.java

```diff
@@ -57,13 +57,15 @@ public static MediaPlayer getMediaPlayerTabletPlacementOverlay(Context context){
     }
 
     public static synchronized void playTabletPlacementOverlay(MediaPlayer mediaPlayerTabletPlacement, MediaPlayer mediaPlayerTabletPlacementOverlay, MediaPlayer mediaPlayerAnimalSound){
-        if (mediaPlayerTabletPlacement != null){
-            if ((!mediaPlayerTabletPlacement.isPlaying()) && (!mediaPlayerTabletPlacementOverlay.isPlaying()) && (!mediaPlayerAnimalSound.isPlaying())){
-                mediaPlayerTabletPlacementOverlay.start();
-            }
-        } else {
-            if ((!mediaPlayerTabletPlacementOverlay.isPlaying()) && (!mediaPlayerAnimalSound.isPlaying())){
-                mediaPlayerTabletPlacementOverlay.start();
+        if ((mediaPlayerTabletPlacementOverlay != null) && (mediaPlayerAnimalSound != null)){
```

Comment:
@sladomic Can you add some comments to the code about what's happening in this method?